addons/stock/models/stock_picking.py

```diff
@@ -921,6 +921,17 @@ def _set_scheduled_date(self):
                 raise UserError(_("You cannot change the Scheduled Date on a done or cancelled transfer."))
             picking.move_ids.write({'date': picking.scheduled_date})
 
+    def onchange(self, values, field_names, fields_spec):
+        """
+        Prevent the picking's scheduled_date from being updated when a move's date is modified, as it is computed when the form view is saved.
+        """
```

Comment:
```suggestion
        """Prevent the picking's scheduled_date from being updated when a move's date is modified,
        as it is computed when the form view is saved.
        """
```
Is the correct way to write docstring. Check https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html or model.py